cmd/loom/gateway/general_cmds.go

```diff
@@ -646,3 +650,52 @@ func newSetWithdrawFeeCommand() *cobra.Command {
 	}
 	return cmd
 }
+
+func newUpdateMainnetGatewayAddressCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update-mainnet-address <mainnet-address> <gateway-name>",
+		Short:   "Update mainet gateway address. Only callable by current gateway owner",
+		Example: updateMainnetAddressCmdExample,
+		Args:    cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			loomKeyPath := gatewayCmdFlags.PrivKeyPath
+			hsmPath := gatewayCmdFlags.HSMConfigPath
+			algo := gatewayCmdFlags.Algo
+			signer, err := cli.GetSigner(loomKeyPath, hsmPath, algo)
+			if err != nil {
+				return err
+			}
+
+			mainnetAddress, err := hexToLoomAddress(args[0])
```

Comment:
It's not a hex address for Binance is it?

Reply:
it can be converted to hex, but for convenient purpose we can check and convert if it was `bech32`

Reply:
It should be in whatever encoding we use in the Binance Oracle config file, if the wallet address is hex then keep this hex, if it's bech32 then convert it here.